source/code/core/tasks/private/task_thread_v2.cxx

```diff
@@ -115,6 +124,11 @@ namespace ice
                         expected_initial_task = next_op;
                     }
                 }
+                else
+                {
+                    // TODO: Remove the sleep once we get to the point of not killing the CPU looping here as crazy.
```

Comment:
Remove 'Sleep' call from `task_thead_v2::run`

Reply:
We might keep it for now, and extend the API to have access to simmilar functions across platforms.